package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Shows a status of the ongoing projects being done",
   "main": "index.js",
   "scripts": {
-    "dev": "next src",
-    "build": "next build src dist",
```

Comment:
I am not sure why did we change the `dev & build` commands previously, so I just changed it to default.
@ankushdharkar let me know if we need to revert back to these.